components/tikv_util/src/metrics/mod.rs

```diff
@@ -36,16 +46,63 @@ mod metrics_reader;
 
 pub type RecordPairVec = Vec<pdpb::RecordPair>;
 
-pub fn dump(should_simplify: bool) -> String {
+lazy_static! {
+    // the registry for high priority metrics.
+    pub static ref HIGH_PRIORITY_REGISTRY: Registry = Registry::new();
+}
+
+static METRICS_COMPACT_LEVZEL: AtomicU32 = AtomicU32::new(MetricsCompactLevel::NoCompact as u32);
+static METRICS_LEVEL: AtomicU32 = AtomicU32::new(MetricsLevel::All as u32);
+
+// reduce the return frequency of normal metrics factor
+const NORMAL_METRICS_REDUCE_FACTOR: u64 = 2;
+static METRICS_REQUEST_COUNTER: AtomicU64 = AtomicU64::new(0);
+
+pub fn set_metrics_compact_level(level: MetricsCompactLevel) {
+    METRICS_COMPACT_LEVZEL.store(level as u32, Ordering::Relaxed);
```

Comment:
maybe Ordering::Release?

Reply:
I originally thought the order is not essential here because we don't need that strong consistency. I will change to `Release` order to make it more  ergonomic.